api/src/main/java/org/geysermc/geyser/api/extension/exception/InvalidExtensionException.java

```diff
@@ -25,10 +25,15 @@
 
 package org.geysermc.geyser.api.extension.exception;
 
+import java.io.Serial;
+
 /**
  * Thrown when an extension is invalid.
  */
 public class InvalidExtensionException extends Exception {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
     public InvalidExtensionException(Throwable cause) {
```

Comment:
space

Reply:
Fixed in https://github.com/GeyserMC/Geyser/pull/4238/commits/a13d42d035e8e9865dcb279e8446d1290e34ec74